evasquez26/angular4-startup
# S2-ONE-44 Create a course functionality

## Story / Task

[ONE-44](https://bixlabs.atlassian.net/browse/ONE-44)


## What this PR does?

Adds the create course functionality completed, (we were missing the enrollDuration and cover photo)


### Before 

- We could not create a course because we could not provide a cover photo and enrollDuration

### After

- We can create a course specifying the cover photo and enrollDuration

### How to test / reproduce

- Login into the app
- Navigate to "My courses section"
- Observe that there's a create option
- Clicking the option should take you to create course screen
- Specifying required fields shoud create the course metadata
- After that you should be redirected to the course edition
- All fields are required and should give you an error if you do not specify them

### TODOs
- [ ] Fix CORS error when uploading an image (backend)
- [ ] Spinner when the image is being uploaded to amazon

### Related PRs

 N/A

## Migrations

NO

Thank you!
